### DISCUSSION TITLE
How to disable "Create new Tenant" option for a user.

### DISCUSSION THREAD

**Discussion by piyushdongre-dot-com** [filamentphp/filament#13916] [Help]
### Package

Panel builder

### Package Version

v3.2.57

### How can we help you?

Hi,
I am assigning each user to a default Tenant, and only Admin can assign them to any new Tenant. I dont want to give user an option to create a new Tenant by themself. How can I not show the option "Create new Tenant" in admin panel.

**Comment by leandrocfe** [comment]
Do you mean `$panel->tenantRegistration()`, right?
Maybe you could create a middleware to conditionally display this page based on the user’s role

```php
public function handle(Request $request, Closure $next): Response
{
    $panel = filament()->getCurrentPanel();

    if(auth()->user()->is_admin) {
        $panel->tenantRegistration(RegisterTeam::class);
    }

    return $next($request);
}
```
